source/includes/kubernetes_extension/_k8_secrets.md

```diff
@@ -96,3 +96,30 @@ Retrieve a secret and all its info in a given [environment](#administration-envi
 | `metadata` <br/>_object_   | The metadata of the secret.                                                |
 | `kind` <br/>_string_       | A string value representing the REST resource that this object represents. |
 | `type` <br/>_string_       | A string used to facilitate programmatic handling of a secret's data.      |
+
+<!-------------------- DELETE SECRET -------------------->
+
+#### Delete a secret
+
+```shell
+curl -X DELETE \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/secrets/default-token-xxxmt/default"
```

Comment:
the ones in the `kubernetes_extensions` need the cluster_id as a query param right?

Reply:
thanks 🤦